src/aiofiles/__init__.py

```diff
@@ -1,5 +1,7 @@
+# -*- coding: utf-8 -*-
 """Utilities for asyncio-friendly file handling."""
-from .threadpool import open
-from . import tempfile
+from aiofiles import tempfile
```

Comment:
Let's use relative imports for components from this package (i.e. not third-party).